src/main/java/HttpServer/ResponseWriter.java

```diff
@@ -36,4 +39,16 @@ public void write() throws IOException {
         System.out.println("<CRLF>");
         writing.writeLine("");
     }
+
+    private String requestLine(HashMap request) throws IOException {
```

Comment:
This function looks like it is creating a response, why is it named `requestLine` ?

Reply:
Because I confused "status line" with "request line." Will change

Reply:
:white_check_mark: